ubid.rb

```diff
@@ -66,6 +66,7 @@ class UBID
   TYPE_DNS_RECORD = "dr"
   TYPE_DNS_SERVER = "ds"
   TYPE_FIREWALL_RULE = "fr"
+  TYPE_SPDK_INSTALLATION = "s1"
```

Comment:
Do we need ubid decoding for everything, or should we have a special prefix for opaque identifiers of the time-based or entropy-based type?

Considerations include:

* Do you anticipate using `udec` a lot?
* Will it ever be an external identifier?
* Will it ever be in a hostname?

Reply:
I added an entropy-based type `TYPE_ETC` to be used for this & other types that we don't usually use in logs & don't need to decode.